style.css

```diff
@@ -266,6 +267,57 @@ a {
   color: #5e6c84;
 }
 
+.popup {
+  display: none;
+  flex-direction: column;
+  gap: 1.2rem;
+  position: fixed;
+  inset: 0 0 0 0;
+  padding: 1.6rem;
+  background-color: #fff;
+  z-index: 1000;
+  overflow-y: auto;
+  height: 100vh;
+}
```

Comment:
Well done 🎉 on the popup, it shows how much you understand DOM interactions but in order to make the design even better, consider making the content of the popup take the whole space available in your popup. If it cannot take the whole width of the popup, consider making the popup smaller and blurring the background maybe. 🌫️ See below the difference between the Figma and what you have 👇🏽 

## Figma Design
![Screenshot 2023-02-01 at 08 36 07](https://user-images.githubusercontent.com/45739963/216057903-cc65619b-e790-4761-b8dc-cd3b661283f5.png)

## Your Popup
![Screenshot 2023-02-01 at 08 35 41](https://user-images.githubusercontent.com/45739963/216057816-a3295477-cabc-4f3b-bf50-20db1e19fa2c.png)

I also think the fonts should be bigger but at this point it's just about attributing space better. 😉